std/experimental/allocator/building_blocks/kernighan_ritchie.d

```diff
@@ -9,9 +9,9 @@ version(unittest) import std.conv : text;
 /**
 $(D KRRegion) draws inspiration from the $(LINK2
 std_experimental_allocator_region.html, region allocation strategy) and also the
```

Comment:
Resolve the merge conflict by incorporating #4409: `$(MREF_ALTTEXT region allocation strategy, std,experimental,allocator,building_blocks,region)`.